src/github.com/getlantern/flashlight/client/chained.go

```diff
@@ -82,10 +87,17 @@ func (s *ChainedServerInfo) Dialer() (*balancer.Dialer, error) {
 	}
 	d := chained.NewDialer(ccfg)
 
+	// Is this a trusted proxy that we could use for HTTP traffic?
+	var trusted string
+	if s.Trusted {
```

Comment:
Not sure about it, this would be printed in the logs, I would not like reading that my content is routed over an "untrusted" peer. This legend will help us now that we are testing, but depending on the actual communication strategy we'll probably need to remove this advertising completely.